### ISSUE TITLE
Image upload for organization

### ISSUE THREAD

**Issue by jaydorsey** [rubyforgood/share_christmas#26]
Upload logo for organization. Organization admin should be able to do this

**Comment by jaydorsey** [comment]
Would suggest waiting on this one and skipping the organization admin overall for the demo. Should leave this issue open but deferred until later.

**Comment by AEgan** [comment]
I hacked on this last night and was able to get things configured. I'm not sure if we'll get to implementing this on the frontend but everything is configured to support it, so I'm going to close it
